src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,111 @@
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: $main-background-color;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  transition-duration: $time-transform;
+
+  &__image {
+    object-position: center;
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    height: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    font-weight: 500;
+    color: $bold-font-color;
+    transition-duration: $time-transform;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400px;
```

Comment:
The `font-weight` property should be a numeric value without units. Change `font-weight: 400px;` to `font-weight: 400;`.